forward/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: forward
 description: TMail forward extension
-publish_to: 'none'
+publish_to: "none"
```

Comment:
please revert it

Reply:
Done

Reply:
Looks like you forgot to revert it. @tddang-linagora